packages/composables/src/composables/useBilling/index.ts

```diff
@@ -3,23 +3,34 @@ import {
   useBillingFactory,
   UseBillingParams
 } from '@vue-storefront/core';
-import type { BillingAddress } from '@vue-storefront/vendure-api';
+import type { CreateAddressInput, Order, OrderAddress } from '@vue-storefront/vendure-api';
+import { BILLING_TYPE } from '../../helpers';
 import type {
   UseBillingAddParams as AddParams
 } from '../../types';
+import { useCart } from '../useCart';
 
-const params: UseBillingParams<BillingAddress, AddParams> = {
+const params: UseBillingParams<OrderAddress, AddParams> = {
+  provide() {
+    return {
+      cart: useCart()
```

Comment:
IMO you can have some issues when you will use other composable outside setup function, but let me know if error occur sometimes?

Reply:
There is no error and we are using similar functionality in order integrations as well. Is it because in some cases we are fetching the data that we already have so better approach would be to reuse the data.